octobot_trading/exchanges/rest_exchange.pxd

```diff
@@ -64,6 +64,7 @@ cdef class RestExchange(AbstractExchange):
     cpdef dict parse_order_book_ticker(self, dict order_book_ticker)
     cpdef double parse_timestamp(self, dict data_dict, str timestamp_key, object default_value=*, bint ms=*)
     cpdef str parse_currency(self, str currency)
+    cpdef str parse_order_id(self, dict order)
```

Comment:
is None an ok cython str ?

Reply:
Yes I just tested it and it returns successfully `None`.